Misc/NEWS.d/next/Core and Builtins/2018-07-23-08-18-07.bpo-27195.V-bmFq.rst

```diff
@@ -0,0 +1,2 @@
+Fix crashes in :class:`io.BufferedReader` and :class:`io.BufferedWriter`.
```

Comment:
Please add a tiny bit more details.

The crash is occurred also when call `tolist()`.